kentikapi/internal/cloud/cloud.go

```diff
@@ -0,0 +1,89 @@
+package cloud
+
+import (
+	"context"
+
+	cloudexportpb "github.com/kentik/api-schema-public/gen/go/kentik/cloud_export/v202101beta1"
+	"github.com/kentik/community_sdk_golang/kentikapi/cloud"
+	kentikerrors "github.com/kentik/community_sdk_golang/kentikapi/internal/errors"
+	"github.com/kentik/community_sdk_golang/kentikapi/models"
+	"google.golang.org/grpc"
+)
+
+// API aggregates Cloud Exports API methods.
+type API struct {
+	client cloudexportpb.CloudExportAdminServiceClient
+}
+
+// NewAPI creates new API.
```

Comment:
Is this useless (but harmless) comment forced by a linter?

Reply:
It is not enforced by a linter, but it is a convention in Go to write comment for every exported symbol (https://go.dev/doc/effective_go#commentary). We reduced this convention to "every exported symbol used by SDK users". Those comments are generated into pkg.go.dev documentation and might not be so useless when there is no code visible (only function signature). It looks like this: https://pkg.go.dev/github.com/kentik/community_sdk_golang@v0.2.0/kentikapi#NewClient

This particular function is actually not going to be used by SDK users so is indeed useless